### ISSUE TITLE
Restore `_space_group.crystal_system` validation method

### ISSUE THREAD

**Issue by merkys** [COMCIFS/cif_core#262]
Currently `cif_core.dic` does not contain any validation methods expressed in dREL, although it had prior to 8ca3b549288396eb1afa38b701d42a50f4ec2100. Said commit had removed validation method for `_space_group.crystal_system` as it contained undefined dREL functions `throw` and `alert`. I think it could be restored and transformed into a statement evaluating to a boolean value, as validation methods should per [Spadaccini et al., 2012](https://pubs.acs.org/doi/10.1021/ci300076w).

**Comment by jamesrhester** [comment]
Yes, I feel that embedding a potentially large number of potentially quite long checks in the definition will lead to clutter, as well as there not being a logical home for checks that aren't local to a particular value (e.g. a group of values being inconsistent).
